### PR TITLE
Update chicago-author-date.csl

### PR DESCRIPTION
**Pull request by serbernardo**
Some changes proposal that will improve the Chicago author-date style in portuguese language. (both portuguese-portugal and portuguese-brasil). These changes were made in the code.

1) It would be also great to translate the following:
In line 628: <text value="filed"/>
and line 638  <text value="issued"/>
It would be better to have instead of filed - submetida em
and instead of issued - publicada em
But I guess it's not possible to localize text values, just terms. :(

2) Also I would like to propose a change for the all the languages. This changing is related how the volume in journal articles is shown. 
In this style code the volume number is shown as:   40 (3)
And the Chicago Manual of Style states that has to be 40, no.3

As stated in: https://www.chicagomanualofstyle.org/book/ed17/part2/ch09/psec027.html

And so, I would suggest the following alteration:
   
<if type="article-journal">
        <choose>
          <if variable="volume">
            <text variable="volume" prefix=" "/>
            <group prefix=", ">
              <choose>
                <if variable="issue">
                  <text variable="issue" prefix="no."/>

### CODE REVIEW

**Comment by adam3smith** [comment]
The volume/issue format is intentional, following 15.47. The manual is quite vague here, also suggesting the other format (e.g. in 15.9), but we generally think the former makes more sense for the style.

File: chicago-author-date.csl [hunk 1]
```diff
@@ -277,10 +323,10 @@
         <choose>
           <if variable="volume">
             <text variable="volume" prefix=" "/>
-            <group prefix=" (" suffix=")">
+            <group prefix=", ">
               <choose>
```
Comment (adam3smith) [inline chicago-author-date.csl:327]: As per my comment, please revert this

**Review by adam3smith — COMMENTED** [review]
One question, one change request.
@bwiernik we didn't introduce terms that could be used for "filed" and "issued" (for patents) did we?

File: chicago-author-date.csl [hunk 2]
```diff
@@ -45,6 +45,52 @@
       <term name="translator" form="short">trans.</term>
     </terms>
   </locale>
+  <locale xml:lang="pt">
```
[the remark this answers is not in the document]
Comment (adam3smith) [reply chicago-author-date.csl:48]: And if this applies generally, I think we should change it in the locale: either but pt-PT or both pt ones

[the remark this answers is not in the document]
Comment (serbernardo) [reply chicago-author-date.csl]: The months are in lowercase for both, portuguese and brasilian portuguese. There is just a difference  that is in Portugal May is abreviated mai. and in brasilian portuguese is maio 
Another difference is that "acessed" in pt-Pt is "acedido a" and in pt-BR is "acessado a". So is prefered to have the changes to the pt-PT. 
I have change the locale to pt-PT and removed the volume entry. You are right and thanks for letting me know that ;)

(I have clicked the add review comment... hope that's not wrong)

**Comment by adam3smith** [comment]
Thanks! I moved the months changes into the actual locale. I know that's a bit unfortunate for Mendeley, because they update these quite infrequently, but I don't want to carry around a ton of unnecessary custom code in the style if we have this in the locales correctly.

**Pull request merged**